src/pages/help-centre/dbot.js

```diff
@@ -373,7 +373,13 @@ const SaveStrategy = () => (
             )}
         </Text>
         <ImageWrapper>
-            <img img_name="dbot-save.png" alt={localize('Save')} width="40.7rem" />
+            <img
+                src={SaveBotImage}
+                alt={localize('Save')}
+                width="40.7rem"
```

Comment:
why do we have two width here?

Reply:
the image element width doesn't affect the actual css layout of the image but it prevents a layout shift once the image is loaded: https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/